src/Calculator/App.xaml

```diff
@@ -803,14 +803,14 @@
                                                  Foreground="{ThemeResource SystemControlForegroundAccentBrush}"
                                                  BorderThickness="0"
                                                  AutomationProperties.AutomationId="CalculationResultScrollLeft"
+                                                 AutomationProperties.Name="{utils:ResourceString Name=CalculationResultScrollLeft/[using:Windows.UI.Xaml.Automation]AutomationProperties/Name}"
```

Comment:
Nit: "[using:Windows.UI.Xaml.Automation]" is not needed anymore, maybe we can directly replace "[using:Windows.UI.Xaml.Automation]AutomationProperties" with "AutomationProperties", to shorten the resource key.

Reply:
Yes, I think it would be good to clean up some of these names. But since that will impact the localization process, I think it would be better to make those changes in a separate PR.